viewer/packages/data-providers/src/pointcloud-stylable-object-providers/types.ts

```diff
@@ -6,42 +6,33 @@ import { StylableObject } from './StylableObject';
 import { IShape } from '@reveal/utilities';
 
 import { Box3 } from 'three';
-import { AnnotationsAssetRef } from '@cognite/sdk';
+import { AnnotationsAssetRef, DirectRelationReference } from '@cognite/sdk';
+import { ClassicDataSourceType, DataSourceType } from '../DataSourceType';
+
+/**
+ * @public
+ * CDF Data model instance reference
+ */
+export type DMInstanceRef = DirectRelationReference;
 
 export type CdfPointCloudObjectAnnotation = {
   annotationId: number;
-  asset?: AnnotationsAssetRef;
+  volumeInstanceRef?: DMInstanceRef;
+  asset?: AnnotationsAssetRef | DMInstanceRef;
   region: IShape[];
```

Comment:
I think this could be made cleaner with a type parameter, so that it either contains `annotationId+assetRef` or `volumeInstanceRef+assetRef`. But I'll leave it up to you whether you want to do that now or not